src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -11,6 +11,7 @@
 public enum StandardActions implements Action {
 
     COPY_MORE(Localization.lang("Copy") + "..."),
+    COPY_TO(Localization.lang("Copy to") + "..."),
```

Comment:
I think this ... should be part of the localization string. I would change the above one as well